### PR TITLE
Add Flores Task

### PR DESCRIPTION
**Pull request by gwenzek**
Test plan: 

tested with the mm100-175m baseline model.

The files and instructions are available on https://github.com/gwenzek/flores/tree/master/dynalab (the readme is still WIP)

```
dynalab-cli test -n mm100-175m-fix                                
Config file validated.
Warning: Size of current project folder is more than 2GB. Please consider add large files or folders to exclude
Tarballing the project directory...
Archive the model for torchserve...
Building docker image...
[+] Building 387.1s (21/21) FINISHED                                
 => [internal] load build definition from Dockerfile           0.0s
 => => transferring dockerfile: 1.76kB                         0.0s
 => [internal] load .dockerignore                              0.0s
 => => transferring context: 2B                                0.0s
 => [internal] load metadata for docker.io/library/ubuntu:18.  1.8s
 => CACHED [ 1/16] FROM docker.io/library/ubuntu:18.04@sha256  0.0s
 => [internal] load build context                             68.9s
 => => transferring context: 3.22GB                           68.9s
 => [ 2/16] RUN apt-get update &&     DEBIAN_FRONTEND=nonin  105.0s
 => [ 3/16] RUN update-alternatives --install /usr/bin/python  0.1s
 => [ 4/16] RUN cd /tmp     && curl -O https://bootstrap.pypa  5.4s
 => [ 5/16] RUN python -m pip install --no-cache-dir torchser  7.5s
 => [ 6/16] RUN python -m pip install --no-cache-dir torch==  71.3s 
 => [ 7/16] COPY ./.dynalab/mm100-175m-fix/tmp/dev-docker-ent  0.1s 
 => [ 8/16] RUN chmod +x /usr/local/bin/dev-entrypoint.sh      0.2s 
 => [ 9/16] RUN mkdir -p /opt/ml/model                         0.1s 
 => [10/16] COPY ./.dynalab/mm100-175m-fix/tmp/mm100-175m-fix  3.3s
 => [11/16] RUN mkdir -p /home/model-server/ && mkdir -p /hom  0.1s
 => [12/16] ADD ./.dynalab/mm100-175m-fix/tmp/mm100-175m-fix.  5.7s
 => [13/16] WORKDIR /home/model-server/code                    0.0s
 => [14/16] RUN python -m pip install -e dynalab               5.1s
 => [15/16] RUN if [ -f requirements.txt ] && [ True = True  165.3s
 => [16/16] RUN if [ -f setup.py ] && [ False = True ]; then   0.1s
 => exporting to image                                        15.7s
 => => exporting layers                                       15.7s
 => => writing image sha256:8af41e1dd0ef5d91faec0daf8d1783970  0.0s
 => => naming to docker.io/library/mm100-175m-fix              0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
Integrated test passed. Torchserve log can be found at ./.dynalab/mm100-175m-fix/tmp/ts_log.err
```

### CODE REVIEW

File: dynalab/tasks/common.py [hunk 1]
```diff
@@ -48,6 +48,10 @@ def _get_mock_context(model_name):
             print("Getting model response ...")
             response = handle_func(mock_data, mock_context)
             print(f"Your model response is {response}")
+            try:
+                json.dumps(response)
+            except Exception as e:
+                raise Exception("The model response isn't serializable to json !") from e
```
Comment (gwenzek) [inline dynalab/tasks/common.py:54]: This allows catching non serializable objects in the response returned by the user.

File: dynalab/tasks/flores_small1.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+class TaskIO(BaseTaskIO):
+    def __init__(self):
+        BaseTaskIO.__init__(self)
+
+    def verify_response(self, response, data):
+        assert "id" in response and response["id"] == data["uid"]
+        assert "translatedText" in response and isinstance(
+            response["translatedText"], str
+        )
+        assert response["signed"] == self.generate_response_signature(response, data)
```
Comment (mazhiyi) [inline dynalab/tasks/flores_small1.py:33]: For security purpose, please follow the convention in other tasks and check there are no extra keys too.

File: dynalab/tasks/common.py (hunk 1 above)
Comment (mazhiyi) [reply dynalab/tasks/common.py:54]: Please raise RuntimeError :)

**Review by mazhiyi — APPROVED** [review]
LGTM for now. Similarly, would be good to a more elaborate test plan (e.g. the printouts in console).

File: dynalab/tasks/common.py (hunk 1 above)
Comment (gwenzek) [reply dynalab/tasks/common.py]: done.

File: dynalab/tasks/flores_small1.py (hunk 2 above)
Comment (gwenzek) [reply dynalab/tasks/flores_small1.py:32]: done.

**Review by gwenzek — COMMENTED** [review]
thanks for the review, I fixed the two comments

**Pull request merged**